### PR TITLE
Option to specify rippled path on command line.

### PR DESCRIPTION
**Pull request by ximinez**
* --rippled=<absolute or relative path>
* Works for "npm test" and "mocha"
* Remove "rippled_path" from config.js to require CLI path.

Reviewers: @howardhinnant, @sublimator

### CODE REVIEW

File: test/server.js [hunk 1]
```diff
@@ -87,7 +87,22 @@ Server.prototype.set_ledger_file = function(fn) {
 Server.prototype._serverSpawnSync = function() {
   var self  = this;
 
-  var path = this.config.rippled_path;
+  var rippledpath = this.config.rippled_path;
+  // Override the config with command line if provided
+  if(process.env["npm_config_rippled"]) {
```
Comment (sublimator) [inline test/server.js]: nice :)

Comment (rec) [inline test/server.js]: Nearly all our Javascript has spaces after the `if` statement...!

**Comment by rec** [comment]
With that quibble, I think this is good to go?

File: test/server.js (hunk 1 above)
Comment (ximinez) [inline test/server.js]: If that's the only problem you can find, I'll go ahead and fix it. :)

On Fri, Jan 2, 2015 at 4:04 PM, Tom Swirly <notifications@github.com> wrote:

> In test/server.js
> <https://github.com/ripple/rippled/pull/778#discussion-diff-22422780>:
>
> > @@ -87,7 +87,22 @@ Server.prototype.set_ledger_file = function(fn) {
> >  Server.prototype._serverSpawnSync = function() {
> >    var self  = this;
> >
> > -  var path = this.config.rippled_path;
> > +  var rippledpath = this.config.rippled_path;
> > +  // Override the config with command line if provided
> > +  if(process.env["npm_config_rippled"]) {
>
> Nearly all our Javascript has spaces after the if statement...!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/rippled/pull/778/files#r22422780>.
>



-- 
*Edward Hennis | *Senior Software Engineer | *Ripple Labs Inc.*
ed@ripple.com | www.ripple.com | *www.ripplelabs.com
<http://www.ripplelabs.com/>*
T. 352.327.8769

**Comment by ximinez** [comment]
Reopened after deleting the wrong branch.

**Comment by sublimator** [comment]
LGTM

**Pull request closed without merging**